projects/ad9081_fmca_ebz_x_band/zcu102/system_bd.tcl

```diff
@@ -166,8 +168,48 @@ create_bd_port -dir O tdd_tx_stingray_en
 
 set tdd_sync_in_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_tdd_0/sync_in]]]
 set tdd_sync_in_pin [get_bd_pins axi_tdd_0/sync_in]
-ad_disconnect $tdd_sync_in_net $tdd_sync_in_pin
+
+set adc_do_m_axis_clk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins $adc_data_offload_name/m_axis_aclk]]]
+set adc_do_m_axis_clk_pin [get_bd_pins $adc_data_offload_name/m_axis_aclk]
+
+set adc_dma_s_axis_clk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_mxfe_rx_dma/s_axis_aclk]]]
+set adc_dma_s_axis_clk_pin [get_bd_pins axi_mxfe_rx_dma/s_axis_aclk]
+
+set adc_do_m_axis_rst_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins $adc_data_offload_name/m_axis_aresetn]]]
+set adc_do_m_axis_rst_pin [get_bd_pins $adc_data_offload_name/m_axis_aresetn]
+
+set adc_dma_m_dest_rst_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_mxfe_rx_dma/m_dest_axi_aresetn]]]
+set adc_dma_m_dest_rst_pin [get_bd_pins axi_mxfe_rx_dma/m_dest_axi_aresetn]
+
+set hp1_fdp_aclk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins sys_ps8/saxihp1_fpd_aclk]]]
+set hp1_fdp_aclk_pin [get_bd_pins sys_ps8/saxihp1_fpd_aclk]
+
+set axi_hp1_interconnect_aclk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_hp1_interconnect/aclk]]]
+set axi_hp1_interconnect_aclk_pin [get_bd_pins axi_hp1_interconnect/aclk]
+
+ad_disconnect $hp1_fdp_aclk_net              $hp1_fdp_aclk_pin
+ad_disconnect $tdd_sync_in_net               $tdd_sync_in_pin
+ad_disconnect $adc_do_m_axis_clk_net         $adc_do_m_axis_clk_pin
+ad_disconnect $adc_dma_s_axis_clk_net        $adc_dma_s_axis_clk_pin
+ad_disconnect $adc_do_m_axis_rst_net         $adc_do_m_axis_rst_pin
+ad_disconnect $adc_dma_m_dest_rst_net        $adc_dma_m_dest_rst_pin
+ad_disconnect $axi_hp1_interconnect_aclk_net $axi_hp1_interconnect_aclk_pin
+
+ad_ip_parameter axi_hp1_interconnect CONFIG.NUM_CLKS 1
```

Comment:
Instead of disconnecting the 2nd clock and reconfiguring the **axi_hp1_interconnect**, I suggest fixing the following clock connection here:
https://github.com/analogdevicesinc/hdl/blob/065c8f186ef87ff049d279ed5859ee8d97d91808/projects/ad9081_fmca_ebz/common/ad9081_fmca_ebz_bd.tcl#L493
where $sys_cpu_clk should be replaced with $sys_dma_clk.

Reply:
Changing the ad9081_fmca_ebz project sources will require additional testing for all Xilinx carriers. That’s why modifying just the x_band project sources was preferred.

Reply:
I'll open a PR which will fix the interconnect clock after I get the chance to test it on hardware, although I'm pretty sure it works.